src/test/java/org/prebid/server/bidder/improvedigital/ImprovedigitalBidderTest.java

```diff
@@ -5,6 +5,7 @@
 import com.iab.openrtb.request.BidRequest;
```

Comment:
Probably it is time to add `givenBidRequest()` and `givenImp()` to be more in line with project.